### PR TITLE
Upgrade to Akka Streams and HTTP 2.0

### PR DESCRIPTION
**Pull request by fsat**

### CODE REVIEW

**Comment by ktoso** [comment]
Wohoo, almost painless :)
LGTM

**Comment by fsat** [comment]
Thanks @ktoso!

File: src/main/scala/akka/contrib/process/BlockingProcess.scala [hunk 1]
```diff
@@ -124,15 +123,13 @@ class BlockingProcess(
     try {
       val selfRef = context.self
 
-      val stdin = Sink.outputStream(process.getOutputStream)
+      val stdin = StreamConverters.fromOutputStream(process.getOutputStream)
         .mapMaterializedValue(_.andThen { case _ => selfRef ! StdinTerminated })
 
-      val stdout = Source.inputStream(process.getInputStream)
-        .withAttributes(Attributes.inputBuffer(1, 1))
```
Comment (fsat) [inline src/main/scala/akka/contrib/process/BlockingProcess.scala]: Removing `withAttributes` due to the fact that `Source` from `InputStream` now emits data eagerly: https://github.com/akka/akka/issues/19193

File: src/main/scala/akka/contrib/process/BlockingProcess.scala [hunk 2]
```diff
@@ -124,15 +123,13 @@ class BlockingProcess(
...
+      val stdout = StreamConverters.fromInputStream(process.getInputStream)
         .mapMaterializedValue(_.andThen { case _ => selfRef ! StdoutTerminated })
 
-      val stderr = Source.inputStream(process.getErrorStream)
-        .withAttributes(Attributes.inputBuffer(1, 1))
```
Comment (fsat) [inline src/main/scala/akka/contrib/process/BlockingProcess.scala]: Removing `withAttributes` due to the fact that `Source` from `InputStream` now emits data eagerly: https://github.com/akka/akka/issues/19193

File: src/main/scala/akka/contrib/process/BlockingProcess.scala (hunk 1 above)
Comment (huntc) [inline src/main/scala/akka/contrib/process/BlockingProcess.scala]: yay!

**Pull request merged**